src/components/savings/APYPill.js

```diff
@@ -51,30 +51,40 @@ const textProps = {
   steps: [0, 1],
 };
 
-const APYPill = ({ small, value }) => (
-  <Centered style={small ? sx.containerSmall : sx.container}>
-    {small ? (
-      <RadialGradient {...radialGradientProps} borderRadius={21} radius={81} />
-    ) : (
-      <LinearGradient
-        {...linearGradientProps}
-        borderRadius={17}
-        colors={gradientColors}
-        opacity={0.1}
-      />
-    )}
-    <GradientText
-      {...textProps}
-      align="center"
-      angle={false}
-      letterSpacing="roundedTight"
-      size={small ? 'smedium' : 'lmedium'}
-      weight="semibold"
-    >
-      {value}% APY
-    </GradientText>
-  </Centered>
-);
+function APYPill({ small, value }) {
+  return (
+    <Centered style={small ? sx.containerSmall : sx.container}>
+      {small ? (
+        <RadialGradient
+          {...radialGradientProps}
+          borderRadius={21}
+          radius={81}
+        />
+      ) : (
+        <LinearGradient
+          {...linearGradientProps}
+          borderRadius={17}
+          colors={gradientColors}
+          opacity={0.1}
+        />
+      )}
+      {Platform.OS === 'ios' ? (
+        <GradientText
+          {...textProps}
+          align="center"
+          angle={false}
+          letterSpacing="roundedTight"
+          size={small ? 'smedium' : 'lmedium'}
+          weight="semibold"
```

Comment:
Shouldn't we pass all this Props to the <Text> component of Android ? (except for angle)